### ISSUE TITLE
Virtual Desktop Passthrough Not Working

### ISSUE THREAD

**Issue by raeldor** [OpenKneeboard/OpenKneeboard#666]
Hi,

I created a bitmap with my Virtual Desktop passthrough color and added to OpenKneeboard, but it's not doing pass through.  Do you know why this might be?  I thought because it was OpenXR compatible that it should work, but it's just showing the passthrough color.

Thanks!

**Comment by fredemmott** [comment]
Sorry, this isn't an OpenKneeboard feature or something I use. Try virtual desktop's support.

**Comment by raeldor** [comment]
It’s not related to this?…https://github.com/OpenKneeboard/OpenKneeboard/issues/608On Sep 16, 2024, at 6:18 PM, Fred Emmott ***@***.***> wrote:﻿
Closed #666 as not planned.

**Comment by fredemmott** [comment]
No. Virtual desktop's passthrough feature is a virtual desktop feature, not an openkneeboard feature.
